### ISSUE TITLE
Missing LICENSE files for `.icons`, in source distributions on PyPI

### ISSUE THREAD

**Issue by pradyunsg** [squidfunk/mkdocs-material#1803]
__I checked that...__

- [x] ... the documentation does not mention anything about my problem
- [x] ... the problem doesn't occur with the default MkDocs template
- [x] ... the problem is not in any of my customizations (CSS, JS, template)
- [x] ... there are no open or closed issues that are related to my problem

## Description

The LICENSE files for the contents of the `.icons` directory are not included in the source distributions on PyPI.

### Expected behavior

These would be included.

### Actual behavior

They're not included. :)

### Steps to reproduce the bug

1. Download the source tarball from https://pypi.org/project/mkdocs-material/#files.
2. Check the files in the tarball.
3. Notice that there's only 1 LICENSE file (none in the `.icons` directory).

```sh-session
$ wget https://files.pythonhosted.org/packages/32/a9/93be906519987d53ee068e528a7fea8dde7c84b3a3869907da1fd7ef14fa/mkdocs-material-5.4.0.tar.gz
$ tar --list --file=mkdocs-material-5.4.0.tar.gz | grep LICENSE
mkdocs-material-5.4.0/LICENSE
```

### Package versions

* Python: not relevant
* MkDocs: not relevant
* Material: 5.4.0 (and looks like all of 5.x series)

### Project configuration

Not relevant.

### System information

Not relevant.

**Comment by squidfunk** [comment]
Thanks for reporting. We just need to add them to the `MANIFEST` 😊

**Comment by squidfunk** [comment]
Fixed in 0ac34860

**Comment by pradyunsg** [comment]
Consider adding check-manifest to the CI linters list, to catch potential issues like this. :)

**Comment by squidfunk** [comment]
Released as part of 5.5.0.

Just realized too late, that the Material community license was also missing, it was added in ae0b1872. This will be part of the next release which should follow shortly 😊 As this is a non-critical issue, I'm closing it now.
